vmm/src/api/mod.rs

```diff
@@ -495,6 +503,24 @@ pub fn vm_add_disk(
     }
 }
 
+pub fn vm_add_fs(
+    api_evt: EventFd,
+    api_sender: Sender<ApiRequest>,
+    data: Arc<FsConfig>,
+) -> ApiResult<()> {
+    let (response_sender, response_receiver) = channel();
+
+    // Send the VM add-fs request.
+    api_sender
+        .send(ApiRequest::VmAddFs(data, response_sender))
+        .map_err(ApiError::RequestSend)?;
+    api_evt.write(1).map_err(ApiError::EventFdWrite)?;
+
+    response_receiver.recv().map_err(ApiError::ResponseRecv)??;
+
+    Ok(())
+}
```

Comment:
I'm wondering if you think we'll ever need better responses from these requests? I've been surprised to see how bare they are.

Reply:
I think it's acceptable for this endpoint, since I doubt we'll need the PCI device ID for the virtio-fs device. The only things you need for mounting a virtiofs device is the tag and dax status which are both sent in the request.